src/base/utils/misc.cpp

```diff
@@ -581,7 +581,7 @@ QString Utils::Misc::libtorrentVersionString()
 QString Utils::Misc::windowsSystemPath()
 {
     static const QString path = []() -> QString {
-        WCHAR systemPath[64] = {0};
+        WCHAR systemPath[PATH_MAX] = {0};
```

Comment:
What is PATH_MAX?
AFAIK, WinAPI defines MAX_PATH constant.

Reply:
Whoops! 
On my system (Qt with mingw32 compiler on win10) both defines are present and qbittorrent builds normally with any of them. 

In C:\Qt\Tools\mingw530_32\i686-w64-mingw32\include\limits.h
```c++
/*
 * File system limits
 *
 * NOTE: Apparently the actual size of PATH_MAX is 260, but a space is
 *       required for the NUL. TODO: Test?
 * NOTE: PATH_MAX is the POSIX equivalent for Microsoft's MAX_PATH; the two
 *       are semantically identical, with a limit of 259 characters for the
 *       path name, plus one for a terminating NUL, for a total of 260.
 */
#define PATH_MAX	260
```
In C:\Qt\Tools\mingw530_32\i686-w64-mingw32\include\minwindef.h
```c++
#define MAX_PATH 260
```

So obviously I should stick to variant available in VisualStudio, but I don't have VS installed.

Reply:
AppVeyor builds with Visual Studio. It fails due to PATH_MAX.

Reply:
Changed it